src/Component/CompositionContext/CompositionContext.example.md

```diff
@@ -0,0 +1,155 @@
+`CompositionContext` lets you disable fields and editors and/or replace fields with custom components.
+
+In order to compose your own editors, one CompositionContext provider has to be wrapped around the editor(s) to customize. All configurations have to be set within the object passed to the value property of CompositionContext. The corresponding interface is defined in `src/Components/CompositionContext/CompositionContext.tsx`.
```

Comment:
I'd prefer `, one CompositionContext` -> `, a CompositionContext` here.

Reply:
done in https://github.com/geostyler/geostyler/pull/1203/commits/f93c32d277513ce22489c84946cf61e30bb090c0